### ISSUE TITLE
Minimal theme is deprecated

### ISSUE THREAD

**Issue by Edholm** [zimfw/zimfw#78]
It seems the [minimal theme](https://github.com/S1cK94/minimal) is deprecated in favour of a new [minimal theme](https://github.com/subnixr/minimal)

The new theme looks nice, any chance of getting it integrated into zim?

//Emil

**Comment by Eriner** [comment]
The "new" minimal theme is seemingly very new and has a completely different look (and likely new implementation, however I have not looked). Let us give it some time to see what happens. While the current theme has been deprecated, I have not noticed any overt problems (and if there are, we can always patch it directly).

**Comment by PatTheMav** [comment]
Personally I prefer the now "deprecated" minimal theme over the new one, so I'd say that retaining the old version and maybe adopting the new one as seperate choices might be the way to go.

**Comment by ericbn** [comment]
Adding to the discussion: we don't use the [original minimal theme](https://github.com/S1cK94/minimal) code anymore, but instead we have a [customized version](https://github.com/Eriner/zim/blob/master/modules/prompt/themes/minimal.zsh-theme) of that same theme that makes use of Zim modules (`git-info`) and functions (`short_pwd`).

**Comment by PatTheMav** [comment]
As I've moved to the new minimal prompt quite a while ago and moved my custom zsh stuff into the `custom` module, I've also customized it to use `git-info` as well. The newer version has a different pwd shortener, so I couldn't use `short_pwd` but it seems to work overall.

The newer version uses a lot of ANSI codes (it even uses a number to set the only highlight color in use), so I haven't put a lot of work into cleaning up those.

If you're interested I could create a PR for that first version - any suggestions for the name? Minimal_2? Minimal_New? Also would I need to include the zsh-theme as well as the setup symlink?
